va/va_compat.c

```diff
@@ -40,4 +40,9 @@ vaCreateSurfaces_0_32_0(
     return vaCreateSurfaces(dpy, format, width, height, surfaces, num_surfaces,
                             NULL, 0);
 }
-VA_CPP_HELPER_ALIAS(vaCreateSurfaces, 0, 32, 0);
+
+#if defined(_WIN32)
+#pragma comment(linker, "/alternatename:vaCreateSurfaces_0_32_0=vaCreateSurfaces@VA_API_0.32.0")
```

Comment:
I don't think you need this? Especially because this needs name mangling for x86 and Arm64EC.